### DISCUSSION TITLE
Error occured while exporting private key

### DISCUSSION THREAD

**Discussion by tayyipcanbay** [smartcontractkit/full-blockchain-solidity-course-py#1220] [Q&A] [answered]
at the duration 4:02:30 ı get an error like this:
<img width="717" alt="image" src="https://user-images.githubusercontent.com/73071862/157240020-ca96f4d3-c144-4f5e-b9f1-148c0c8741fa.png">

**Comment by cromewar** [comment, marked as answer]
Hello @tayyipcanbay 
Export is for Unix based systems Linux/Mac if you want to add env variables to windows use: windows advanced configuration and add it to path.

**Reply by tayyipcanbay** [reply]
thank you it fixed 👯
